src/js/player.js

```diff
@@ -1632,6 +1632,9 @@ class Player extends Component {
     this.ready(function() {
       if (method in middleware.allowedSetters) {
         return middleware.set(this.middleware_, this.tech_, method, arg);
+
+      } else if (method in middleware.allowedMediators) {
```

Comment:
`player.play()` seems to be done via `techGet` for the html5 tech, but it's conceivable that a future mediator will use `techCall`

Reply:
maybe we should just make a `techCallAndGet` for stuff like play?

Reply:
That might make sense; I don't really strongly about it one way or another.

Reply:
Let's get everything else ready and then come back to this.

Reply:
eh, I think we can just leave it like this.

Reply:
Yeah, doesn't feel like there's a pressing need to change this yet